### PR TITLE
llm-bitsandbytes-qlora cn done

### PR DESCRIPTION
**Pull request by nuass**
@zhongdongy , pls help review, thx.

### CODE REVIEW

**Review by zhongdongy — CHANGES_REQUESTED** [review]
Please update and submit a revised translation. For details, please check out the comments in this PR.

File: zh/llm-bitsandbytes-qlora.md
[the remark this answers is not in the document]
Comment (nuass) [reply zh/llm-bitsandbytes-qlora.md:1]: Thank you for your detailed review. I will revise it accordingly. It is obvious that many of the problems are caused by my failure to follow the original text's markdown format.

**Comment by zhongdongy** [comment]
@nuass Any updates to this PR? 🤗

**Comment by nuass** [comment]
@zhongdongy I don't have enough time to complete this work in the last month. If you can, please temporarily close this pr. Sorry for the delay.

**Pull request closed without merging**